src/overmap.cpp

```diff
@@ -1229,6 +1238,11 @@ oter_id &overmap::ter(const int x, const int y, const int z)
     return layer[z + OVERMAP_DEPTH].terrain[x][y];
 }
 
+oter_id &overmap::ter( const tripoint &p )
```

Comment:
Are you using it as non-`const`? Would be better to scrap the non-`const` getters and use setters instead, to make more proper encapsulation possible.